source/GameData.cpp

```diff
@@ -627,6 +630,13 @@ const Set<TestData> &GameData::TestDataSets()
 
 
 
+ConditionsStore &GameData::GlobalConditions()
```

Comment:
GameData doesn't feel like the best home for the global conditions to me; loading the global conditions in main.cpp and then giving them to PlayerInfo.cpp directly (as a reference) feels more correct. Main.cpp and PlayerInfo.cpp are already involved anyway, and this would skip the in-between step of GameData (and avoids the need to include ConditionsStore in GameData).

But we discussed this on Discord, and keeping those conditions here in GameData is not a blocker for me.